daprdocs/content/en/operations/monitoring/jaeger.md

```diff
@@ -0,0 +1,106 @@
+---
+type: docs
+title: "How-To: Set up Jaeger for distributed tracing"
+linkTitle: "Jaeger"
+weight: 3000
+description: "Set up Jaeger for distributed tracing"
+type: docs
+---
+
+The following steps shows you how to configure Dapr to send distributed tracing data to Jaeger running as a container in your Kubernetes cluster, how to view them.
```

Comment:
Is this only for Kubernetes or can me make it more general (for example if someone wants to run a Jaeger container locally) if it has to be Kubernetes we should add it in the title